ui/src/components/common/damlTypes.ts

```diff
@@ -16,6 +16,8 @@ type DamlTuple<T> = {
     [key: string]: T;
 }
 
+export type ITextMap<T> = { [key: string]: T };
```

Comment:
is this type still needed? I would change it to something like

```typescript
export type ITextMap<T> = { textMap: [key: string]: T };
```

and then specify it as the return type in the `wrapTextMap` function. Also the `I` in front can be dropped because it is not an interface, (though I think I have been guilty of this myself in the past! old habits die hard)